ext/sdk/resources/sdk-root/shell/src/resource-templates/ts/scaffolder.ts

```diff
@@ -0,0 +1,233 @@
+import { inject, injectable } from "inversify";
+import { FsService } from "backend/fs/fs-service";
+import * as cp from 'child_process';
+import { concurrently } from "utils/concurrently";
+import { ResourceTemplateScaffolder, ResourceTemplateScaffolderArgs } from "../types";
+
+@injectable()
+export default class TsScaffolder implements ResourceTemplateScaffolder {
+  @inject(FsService)
+  protected readonly fsService: FsService;
+
+  async scaffold({ request, manifest, resourcePath }: ResourceTemplateScaffolderArgs) {
+    const resourceName = request.assetName;
+    
+    manifest.clientScripts.push('dist/client/*.client.js');
+    manifest.serverScripts.push('dist/server/*.server.js');
+
+    manifest.fxdkWatchCommands.push(['yarn', [
+      'webpack', '--mode', 'development', '--watch'
+    ]]);
+    manifest.fxdkWatchCommands.push(['yarn', [
+      'webpack', '--mode', 'production'
+    ]]);
+    
+    const promises = [];
+    const relativePath = (to: string) => this.fsService.joinPath(resourcePath, to);
+
+
+
+    promises.push(this.fsService.writeFile(relativePath('webpack.config.js'), getWebpackContent()));
+    promises.push(this.fsService.writeFile(relativePath('package.json'), getPackageContent(resourceName)));
+    
+
+    promises.push(
+      this.fsService.mkdirp(relativePath('client'))
+        .then(() => concurrently(
+          this.fsService.writeFile(relativePath('client/client.ts'), getCode('client')),
+          this.fsService.writeFile(relativePath('client/tsconfig.json'), getTsConfigClient())
+        )),
+    );
+
+
+    promises.push(
+      this.fsService.mkdirp(relativePath('server'))
+        .then(() => concurrently(
+          this.fsService.writeFile(relativePath('server/server.ts'), getCode('server')),
+          this.fsService.writeFile(relativePath('server/tsconfig.json'), getTsConfigServer())
+        )),
+    );
+
+    await Promise.all(promises);
+
+    await this.installModules(resourcePath);
+  };
+
+  private async installModules(path: string) {
+    return new Promise<void>((resolve, reject) => {
+      cp.exec('yarn', { cwd: path, windowsHide: true }, (err) => {
+        if (err) {
+          return reject(err);
+        }
+
+        return resolve();
+      })
+    })
+  }
+}
+
+
+function getTsConfigClient() {
+  return `
+{
+  "compilerOptions": {
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "ES2018",
+    "allowJs": true,
+    "lib": ["ES2018"],
+    "types": ["@citizenfx/client", "@types/node"],
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "include": ["./**/*"],
+  "exclude": ["**/node_modules"]
+}
+  `
+}
+
+
+function getTsConfigServer() {
+  return `
+  {
+  "compilerOptions": {
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "ES2018",
+    "allowJs": false,
```

Comment:
Aight, I get that im very nitpicky, but why is it false? :)

Reply:
Uhm..no good answer to that one (must have slipped through). I can change that out. I do agree it should be true 👍